FrontEnd/src/components/profileList/ProfileCard.module.css

```diff
@@ -0,0 +1,125 @@
+.company-card {
+  display: flex;
+  width: 1305px;
+  padding: 8px;
+  align-items: flex-start;
+  gap: 16px;
+  border-radius: 12px;
+  border: 1px solid var(--main-grey-10, #eeeff1);
+  background: var(--main-white, #fff);
+
+  /* Shadows/Small */
+  box-shadow: 0px 4px 8px 0px rgba(65, 64, 69, 0.2);
+}
+
+.logo-box {
+  width: 64px;
+  height: 64px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 32px;
+  box-shadow: 0px 0px 2px 0px rgba(65, 64, 69, 0.2);
+}
+
+.logo {
+  width: 54px;
+  height: 54px;
+  border-radius: 26px;
+}
+.description__avatar {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  border-radius: 100px;
+  border: 3px solid #e2e5eb;
+  background: #fff;
+}
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  flex: 1 0 0;
+  border-radius: 0px 0px 12px 12px;
+}
+
+.content-header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 5px;
+  align-self: stretch;
+}
+
+.content-header__activity {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.content-header__activity--text {
+  display: flex;
+  align-items: center;
+  color: #767f86;
+  font-feature-settings: "calt" off;
+
+  /* Text/16-Semi bold */
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 20px; /* 125% */
+  letter-spacing: -0.16px;
+}
+
+.content-header__name {
+  display: flex;
+  align-items: flex-start;
+  gap: 4px;
+  align-self: stretch;
+  color: var(--main-grey-90, #25292c);
+
+  /* Header/H6 */
+  font-family: Inter, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 120%; /* 24px */
+  text-transform: uppercase;
+}
+
+.content-header__address {
+  display: flex;
+  align-items: flex-start;
+  gap: 4px;
+  color: #292e32;
+  font-feature-settings: "calt" off;
+
+  /* Text/10-Regular */
```

Comment:
I do not really think that we need such comments. Comments should be used carefully, not for explaining such obvious stuff. I would recommend reading the Clean Code by Robert Martin

Reply:
My bad. Removed comments. These ones missed before pushing.